cwms-http-client/src/main/java/mil/army/usace/hec/cwms/http/client/OkHttpClientInstance.java

```diff
@@ -54,9 +54,8 @@ private OkHttpClientInstance() {
 
     // package scoped for testing only
     static OkHttpClient createClient() {
-        if(LOGGER.isLoggable(Level.FINE))
-        {
-            LOGGING_INTERCEPTOR.level(HttpLoggingInterceptor.Level.BASIC);
+        if (LOGGER.isLoggable(Level.FINE)) {
```

Comment:
This should be different levels of logging mapped to FINE, FINER, FINEST

Reply:
Updated so that this is only logging the bodies of POSTs. I have that happening by default, so lmk if you think that should occur under Level.ALL. Or if its fine to keep as default behavior for all POST requests.